src/components/ChatEntry.js

```diff
@@ -3,28 +3,17 @@ import './ChatEntry.css';
 import PropTypes from 'prop-types';
 import TimeStamp from './TimeStamp.js';
 
-const ChatEntry = (props) => {
-  const onLikeClick = () => {
-    const updatedEntry = {
-      id: props.id,
-      sender: props.sender,
-      body: props.body,
-      timeStamp: props.timeStamp,
-      liked: !props.liked,
-    };
-    props.onUpdate(updatedEntry);
-  };
-
+const ChatEntry = ({ id, sender, body, timeStamp, liked, onLikeClicked }) => {
```

Comment:
just showing that this new syntax is equivalent to the old syntax that you were using :) I just learned that this is called "[destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)" the prop variable.